go/vt/vtgate/planbuilder/operators/apply_join.go

```diff
@@ -140,54 +140,80 @@ func (a *ApplyJoin) AddJoinPredicate(ctx *plancontext.PlanningContext, expr sqlp
 	return nil
 }
 
-func (a *ApplyJoin) AddColumn(ctx *plancontext.PlanningContext, expr sqlparser.Expr) (int, error) {
+func (a *ApplyJoin) pushColLeft(ctx *plancontext.PlanningContext, e *sqlparser.AliasedExpr) (int, error) {
+	newLHS, offset, err := a.LHS.AddColumn(ctx, e, true)
+	if err != nil {
+		return 0, err
+	}
+	a.LHS = newLHS
+	return offset, nil
+}
+func (a *ApplyJoin) pushColRight(ctx *plancontext.PlanningContext, e *sqlparser.AliasedExpr) (int, error) {
```

Comment:
Let's add a new line here

```suggestion
}

func (a *ApplyJoin) pushColRight(ctx *plancontext.PlanningContext, e *sqlparser.AliasedExpr) (int, error) {
```